src/components/ledger/ConfirmLedgerTransaction.vue

```diff
@@ -220,10 +220,21 @@ export default class ConfirmLedgerTransaction extends Vue {
 
   @Emit('successConfirmation')
   async toTrackId() {
-    const LEDGER_STATUS_CODES = { TRANSACTION_CANCELLED_BY_USER: 27013, DEVICE_LOCKED: 27010 };
+    const LEDGER_STATUS_CODES = {
+      TRANSACTION_CANCELLED_BY_USER: 27013,
+      DEVICE_LOCKED: 27010,
+      USER_EXITED_APP: 27906,
+    };
```

Comment:
I think this codes should be on the constants file or an static constant in ´LedgerService´

Reply:
Updated thank you.